### PR TITLE
QingMR *: Optimize manual and billing

### PR DESCRIPTION
**Pull request by lijuan777**
What type of PR is this?
/Optimize

Which issue(s) this PR fixes?
NA

What this PR does / why we need it ?
Summary: 
1. Add quickstart of spark, flink, hive and ranger
2. Add billing document
3. rebuild cluster and node manual

Special notes for your reviewer?
NA

/assign @pethu1110

### CODE REVIEW

File: content/bigdata/qingmr/billing/change_price.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+在您使用 QingMR 过程中，以下场景将会发生费用变更：
+
+- 变更配置：包括增加节点、扩容节点规格。
+- 变更计费模式：从按需计费变更为包年/包月，或从包年/包月变更为按需计费。
+
+## 变更配置
+
+在创建 QingMR 集群时，有多种资源类型供您选择，您可以根据业务实际情况选择合适的节点资源。当集群创建成功后，您也可以对节点数量和规格进行变更。变更配置成功后，将会按照变更后的配置进行计费。   
+- [新增节点](../../manual/mgt_node/add_node)：根据增加的节点个数和不同角色节点的规格，实例资源的费用也会相应增加。
```
Comment (molliezhang) [inline content/bigdata/qingmr/billing/change_price.md:18]: 可修改为 **增删节点** 说明

File: content/bigdata/qingmr/billing/change_price.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [扩容节点](../../manual/mgt_node/capacity_expansion)：增加已有节点的 CPU、内存和硬盘容量，会根据调整之后的实例资源规格收取相应的费用。
+
+> **说明**
+> 
+> 对于包年/包月的集群，变更配置不会改变集群到期时间。变更配置后，增加的费用为：从当前时间到集群到期时间之间的费用。
```
Comment (molliezhang) [inline content/bigdata/qingmr/billing/change_price.md:23]: 补充 包年/包月的集群 **删除节点**，费用变化情况

补充不支持缩容

File: content/bigdata/qingmr/billing/change_price.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## 切换计费模式
+
+[切换计费模式](../../manual/mgt_cluster/switch_billing_mode)有如下两种场景：
+
+- 当前计费模式为按需计费：按需计费模式切换为包年/包月模式，切换后立即生效，可以享受更大的价格优惠。
+- 当前计费模式为包年/包月：包年/包月模式执行切换计费模式操作后，不会立即切换计费模式，集群到期后切换的计费模式才生效。
+
```
Comment (molliezhang) [inline content/bigdata/qingmr/billing/change_price.md:31]: 补充 **变更云服务器类型**

File: content/bigdata/qingmr/billing/price.md [hunk 4]
```diff
@@ -1,43 +1,38 @@
...
-description: 本小节主要介绍 QingCloud QingMR  计费说明。 
-keywords: QingMR  计费说明, 
+description: 本小节主要介绍 QingCloud QingMR 计费说明。 
+keywords: 云计算, 大数据, 青云, QingCloud, QingMR, 计费说明
 weight: 10
 collapsible: false
 draft: false
 ---
 
+## 费用说明
```
Comment (molliezhang) [inline content/bigdata/qingmr/billing/price.md:10]: 列表形式展示 **计费项**

**Review by pethu1110 — APPROVED** [review]

**Pull request merged**